apollo-ios-codegen/Sources/IR/IR+ScopeDescriptor.swift

```diff
@@ -3,9 +3,6 @@ import OrderedCollections
 import GraphQLCompiler
 import Utilities
 
-// TODO: Write tests that two inline fragments with same type and inclusion conditions,
```

Comment:
I believe this is covered by the `test__deferredFragments__givenSiblingDeferredInlineFragmentsOnSameTypeCase_doesNotMergeDeferredFragments` test in [IRRootFieldBuilderTests (line 5142)](https://github.com/apollographql/apollo-ios-dev/blob/feature/defer-execution-networking/Tests/ApolloCodegenTests/CodeGenIR/IRRootFieldBuilderTests.swift#L5142).